charts/tempo-distributed/values.yaml

```diff
@@ -2166,6 +2235,9 @@ adminApi:
     initialDelaySeconds: 45
 
   resources:
+    limits:
+      cpu: 10m
+      memory: 32Mi
```

Comment:
This is almost certainly too small.  Perhaps we leave this as a comment for folks to use in their own values.

Reply:
Yes, since I hadn't used this in the past, I did not change the default values.